alembic/versions/2024_10_17_3503b53a9bc3_make_pacbio_attributes_non_nullable_.py

```diff
@@ -0,0 +1,180 @@
+"""Make all Pacbio model attributes non-nullable
+
+Revision ID: 3503b53a9bc3
+Revises: fd7c9d246255
+Create Date: 2024-10-17 10:24:44.124036
+
+"""
+
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "3503b53a9bc3"
+down_revision = "fd7c9d246255"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.alter_column(
+        table_name="pacbio_sequencing_run",
+        column_name="run_name",
+        existing_type=mysql.VARCHAR(length=64),
+        type_=sa.String(length=32),
```

Comment:
Is there a reason for changing the type here?

Reply:
This was detected by alembic `autogenerate` and I left it as I found it relevant. In the model, this attribute is defined as `Mapped[Str32]` but it was originally defined in the database as `mysql.VARCHAR(length=64)`. Not a big deal but it is more correct now

Reply:
resolved in person